kamon-core/src/main/scala/kamon/metric/TraceMetrics.scala

```diff
@@ -24,6 +24,7 @@ class TraceMetrics(instrumentFactory: InstrumentFactory) extends GenericEntityRe
    *  Records blah blah
    */
   val elapsedTime = histogram("elapsed-time", unitOfMeasurement = Time.Nanoseconds)
+  val traceErrors = counter("trace-errors")
```

Comment:
the name should be just `errors`, as it is already in a trace entity, same bellow for the segments.